shell/platform/windows/window_win32.cc

```diff
@@ -95,6 +95,17 @@ void WindowWin32::InitializeChild(const char* title,
     OutputDebugString(message);
     LocalFree(message);
   }
+  DEVMODE dmi;
+  ZeroMemory(&dmi, sizeof(dmi));
+  dmi.dmSize = sizeof(dmi);
+  if (EnumDisplaySettings(NULL, ENUM_CURRENT_SETTINGS, &dmi)) {
+    framerate_ = dmi.dmDisplayFrequency;
+    SetTimer(result, kVsyncTimer, 1000 / framerate_, nullptr);
```

Comment:
It looks like we're replicate vsync signal logic here, except less accurately since this will round to 16. Is there a reason we couldn't get this from `FlutterWindowsEngine::OnVsync` instead, that way we keep a single implementation?

Reply:
It's my understanding that OnVsync runs only when a frame needs to be scheduled. This is more like a poll for input situation, the gesture data from this timer might be what's necessary to cause a frame to be scheduled. Maybe I should rename it, I just used the framerate since that's the reasonable frequency to do input polling at for general purpose applications.

Reply:
Ah yep - you're entirely right.